tests/phpunit/tests/IXR/IXR-message.php

```diff
@@ -0,0 +1,23 @@
+<?php
+/**
+ * Unit tests covering IXR_Message functionality.
+ *
+ * @package    WordPress
+ * @subpackage IXR
+ */
+
+/**
+ * Test wp-includes/IXR/class-IXR-message.php
+ *
+ * @group IXR
+ */
+class Tests_IXR_IXR_Message extends WP_UnitTestCase {
+
+	public function test_parse() {
```

Comment:
```suggestion
	/**
	 * @ticket 56033
	 *
	 * @covers IXR_Message::tag_open
	 */    
	public function test_tag_open_does_not_create_dynamic_property() {
```

Reply:
I don't really care for the `@covers`, this doesn't cover that method any more particularly that it covers the `parse` method it directly calls. But I don't care enough to argue over it.

Reply:
Well, you could just add `@coversNothing` if you prefer, but there should be some sort of `covers` tag.

Reply:
I'll go with whichever you and other Core devs prefer. Feel free to update the PR directly if you want.